frontend/packages/operator-lifecycle-manager/src/components/package-manifest.tsx

```diff
@@ -24,29 +26,29 @@ const tableColumnClasses = [
   '',
 ];
 
-export const PackageManifestTableHeader = () => [
+export const PackageManifestTableHeader = (t: TFunction) => () => [
   {
-    title: 'Name',
+    title: t('public~Name'),
     sortFunc: 'sortPackageManifestByDefaultChannelName',
     transforms: [sortable],
     props: { className: tableColumnClasses[0] },
   },
   {
-    title: 'Latest version',
+    title: t('public~Latest version'),
     props: { className: tableColumnClasses[1] },
   },
   {
-    title: 'Created',
+    title: t('public~Created'),
     sortField: 'metadata.creationTimestamp',
     transforms: [sortable],
     props: { className: tableColumnClasses[2] },
   },
 ];
 
-export const PackageManifestTableHeaderWithCatalogSource = () => [
-  ...PackageManifestTableHeader(),
+export const PackageManifestTableHeaderWithCatalogSource = (t: TFunction) => () => [
```

Comment:
Avoid passing `t` to a function. Better to call `useTranslation` in the case of component or just use the global instance outside of a component.